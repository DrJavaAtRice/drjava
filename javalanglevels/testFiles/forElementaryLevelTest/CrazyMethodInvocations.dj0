class CrazyMethodInvocations extends Object{
  int method1() {
    return 5;
  }
  
  int method2(int i) {
    return i + 7;
  }
  
  int method3(int i, int j) {
    return i*j+42;
  }
  
  int method4(int i, int j, int k) {
    return i+j+k*5;
  }
  
  int mainMethod() {
    int h = method1();
    int i = method2(h);
    int j = method3(h,i);
    int k = method4(h,i,j);
    return k;
  }
  
}