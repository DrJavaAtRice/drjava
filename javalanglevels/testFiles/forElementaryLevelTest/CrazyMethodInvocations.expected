// Language Level Converter line number map: dj*->java. Entries: 26
//     1->6         2->7         3->8         4->9         5->10        6->11        7->12        8->13   
//     9->14       10->15       11->16       12->17       13->18       14->19       15->20       16->21   
//    17->22       18->23       19->24       20->25       21->26       22->27       23->28       24->29   
//    25->30       26->31   
class CrazyMethodInvocations extends Object{
  public int method1() {
    return 5;
  }
  
  public int method2(final int i) {
    return i + 7;
  }
  
  public int method3(final int i, final int j) {
    return i*j+42;
  }
  
  public int method4(final int i, final int j, final int k) {
    return i+j+k*5;
  }
  
  public int mainMethod() {
    final int h = method1();
    final int i = method2(h);
    final int j = method3(h,i);
    final int k = method4(h,i,j);
    return k;
  }
  

  /** This method is automatically generated by the Language Level Converter. */
  public CrazyMethodInvocations() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      CrazyMethodInvocations cast = ((CrazyMethodInvocations) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}
